daemon/algod/api/algod.oas2.json

```
@@ -1365,6 +1366,10 @@
           "description": "\\[algo\\] total number of MicroAlgos in the account",
           "type": "integer"
         },
+        "min-balance": {
+          "description": "\\[algo\\] total number of MicroAlgos needed to allow transacting from account",
```

Suggestions for better wording greatly appreciated

---

The min-balance isn't just for transacting. I'd phrase it as :
```
total number of MicroAlgos required for maintaining the account
```

---

👍

---

The square brackets aren't needed for this field, the value in the brackets is the msgpack encoded name.
```suggestion
          "description": "MicroAlgo balance required by the account, the requirement starts at 1000 and grows based on the asset and application usage",
```